caddyhttp/browse/browse_test.go

```diff
@@ -257,6 +272,9 @@ func TestBrowseJson(t *testing.T) {
 			Mode:    f.Mode(),
 		})
 	}
+
+	// Test that sort=name returns correct listing, as per filesystem
```

Comment:
That actually not *as per filesystem*, which usually doesn't guarantee any order. Remove everything after the comma.